voskcli/transcribe.py

```diff
@@ -212,7 +212,70 @@ def transcribe(inputFile, outputFile, model):
           f'of {avg_confidence:.2f}. Saving WebVTT file...')
     vtt.save(outputFile)
     print('WebVTT saved.')
-    # print(vtt.content)
+
+
+def detect_model(inputFile, probeTime):
+    '''
+    Detect best model to use.
+
+    :param inputFile: Path to input file
+    :type inputFile: str
+    :param probeTime: Probe time in seconds
+    :type probeTime: int
+    :return: Best model to use
+    :type model: str
+    '''
+    available_models = glob('./models/*') + glob('/usr/share/vosk/models/*')
+    best_model = None
+    best_confidence = 0
+
+    # Get input duration
+    command = ['ffprobe', '-loglevel', 'quiet', '-show_format', '-of', 'json',
```

Comment:
It's usually included in the packaging for FFmprg (e.g. it's part of the FFmpeg RPMs or the official static builds). Of course, that's no guarantee. I'll list it under dependencies in the main readme.

Reply:
Updated this.

You can now specify multiple models (`-m model1 model2``) or languages (`-l lang1 lang2`) or use the special value `auto`. The latter should be especially handy for the Opencast integration since even with the current service/woh you should already be able to just specify the language as `auto`.